src/components/cards/CollaborationsCard.jsx

```diff
@@ -134,25 +132,15 @@ export default function CollaborationsCard({ userId }) {
           setCollabDialogButtonClickLoading
         }
       />
-      {loading && (
-        <LinearProgress style={{ marginTop: 24, marginBottom: 8 }} />
-      )}
-      {tableData.length === 0 ? (
-        <Text
-          variant="body2"
-          id="NO_COLLABORATIONS"
-          style={{ marginTop: 12 }}
-        />
-      ) : (
-        <DataDisplay
-          loading={loading || collabDialogButtonClickLoading}
```

Comment:
Why did this not work lol.

Reply:
It didn't work because if `tableData.length` was zero, then the table component wasn't being rendered at all. (While the table component is loading there is no data).